src/components/Layout/index.ts

```diff
@@ -0,0 +1 @@
+export * from './Layout';
```

Comment:
would you still need this file, since Layout seems like one level and  its the same as `src/components/index.ts`

Reply:
Yes. When you import a folder it would automatically imports the index.{js,ts} file. This is how you import multiple components with a single import while in different folders. 

```ts
import {
  DevfileGalleryFilter,
  DevfileGalleryGrid,
  DevfileGallerySearchBar,
  ErrorBanner
} from '@src/components';
```